### ISSUE TITLE
`Payments > Settings` fee tooltips are confusing, not clear which fees apply to which transactions

### ISSUE THREAD

**Issue by aheckler** [Automattic/woocommerce-payments#9312]
p1724347303581489-slack-CGGCLBN58

In the Slack discussion above, @dechov noted how our current fee tooltip isn't ideal.

![image](https://uploads.linear.app/f629fcde-6f9b-48e8-8af3-60eed8823bff/6568e727-7c42-4713-b460-55a238fda66d/bc32935c-5e68-4123-86f9-0e7c915a1d91?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2Y2MjlmY2RlLTZmOWItNDhlOC04YWYzLTYwZWVkODgyM2JmZi82NTY4ZTcyNy03YzQyLTQ3MTMtYjQ2MC01NWEyMzhmZGE2NmQvYmMzMjkzNWMtNWU2OC00MTIzLTg2ZjktMGU3YzkxNWExZDkxIiwiaWF0IjoxNzg1NzY4ODI4LCJleHAiOjE3ODU3NjkxMjh9.BaUkhvzTTgAz1qZrtERpHDFZZMGpvJ0R5mdNHMDDUhI)

> IMO this payment method tooltip implies that those fees might all be charged on every transaction

This tooltip can be located in the Payments > Settings section. Hovering over the `From x.x% + ...` pricing pills will expand and display this tooltip.

The main issue is the tooltip (as a whole) doesn't clearly indicate what is *always* charged vs. what *might* be charged under specific conditions.

A part of this is the `Total per transaction` line, which indicates the maximum fees that can be charged on a per-transaction basis. However, the current tooltip design/copy doesn't provide any indication that these additional fees will not always be charged (e.g., domestic transactions with no FX).

Additionally, we should also avoid capitalizing "Fees", so change the last line to:

> Learn more about WooPayments fees in your country

(keeping the link intact of course)

---

**Scope (updated Aug 2026)**

This issue is now scoped to the tooltip only. Pill removal and showing the help (?) icon only on activated payment methods have been split into a separate issue, [WOOPMNT-6339](https://linear.app/a8c/issue/WOOPMNT-6339/payments-settings-remove-fee-pills-show-fee-help-icon-only-on).

Implement the tooltip in line with this design, which separates fees that always apply from those that only *may* apply:

![image](https://uploads.linear.app/f629fcde-6f9b-48e8-8af3-60eed8823bff/45a6fcfb-ac9a-44c6-b4fc-87273d54a078/7a5d24aa-9a53-421d-aa66-031fcc859b51?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2Y2MjlmY2RlLTZmOWItNDhlOC04YWYzLTYwZWVkODgyM2JmZi80NWE2ZmNmYi1hYzlhLTQ0YzYtYjRmYy04NzI3M2Q1NGEwNzgvN2E1ZDI0YWEtOWE1My00MjFkLWFhNjYtMDMxZmNjODU5YjUxIiwiaWF0IjoxNzg1NzY4ODI4LCJleHAiOjE3ODU3NjkxMjh9.a857afExHmjp4QCFO5BAO5ta0La6NaZ4gtSPmby8BnQ)

Acceptance criteria:

* Fees that apply to every transaction are visually separated from conditional ones (international payment method fee, currency conversion fee).
* **The** `Total per transaction` **line is removed.** Not relabelled — removed. In the design the conditionality is carried by structure instead: a shaded `Base fee` block, then an `Additional fees that may apply:` heading above a second block containing the conditional fees. There is no summed maximum anywhere in the tooltip.
* The tooltip gains a `Fees details` heading, per the design.
* Link text reads "Learn more about WooPayments fees in your country" (lowercase "fees"), link target unchanged.
* **The pill and the tooltip do not contradict each other.** Today the pill reads `From 2.32% + $0.24` while the tooltip totals `4.52% + $0.24`, and a merchant cannot tell which is the real rate. After this change, the figure on the pill and the figure the tooltip leads with should refer to the same thing, so someone reading both sees one consistent rate. Per @lucyneb: *"we should definitely make sure the pill information matches the content in the current tooltip."* Note this is about the two not disagreeing — removing the pill entirely stays out of scope (see [WOOPMNT-6339](https://linear.app/a8c/issue/WOOPMNT-6339/payments-settings-remove-fee-pills-show-fee-help-icon-only-on)).

Full designs and background: p6riRB-b7J

**Comment by haszari** [comment]
@souravdebnath1986 @rogermattic  – any feedback or suggestions on proposed changes?

**Comment by csmcneill** [comment]
Moving Adam's recommendation here:

> We should consider removing the "Total per transaction" line and replacing it with the dispute fee, like so:
> 
> ![image](https://github.com/user-attachments/assets/185b74ba-b121-4dd5-a00a-bf1a8c593e67)

**Comment by haszari** [comment]
> A part of this is the Total per transaction line, which indicates the maximum fees that can be charged on a per-transaction basis. However, the current tooltip design/copy doesn't provide any indication that these additional fees will not always be charged (e.g., domestic transactions with no FX).

The problem seems to be that the information hierarchy isn't clear:

- These are the fees/costs applied to all transactions
- These are some additional fees/costs that apply in certain circumstances
- This is the total max transaction cost you will pay, if all the above things apply to that transaction

We could address this problem in a few ways:

- Use a heading, box, or other hierarchy to fence off the sometimes additional costs. `If applicable` similar wording, grey or de-emphasised?
- Express the total fees as a range – min and max
  - We have this now, the top line `Base fee` and the bottom line, but it's not clear
- Changing the label `Total per transaction` to something clearer
  - Use the word "maximum" or similar 
  - Use consistent wording in top `Base fee` and bottom `Total`/max fee line, e.g.
    - `Base transaction fee: 2.9% + 0.30`
    - … some info about other additional fees apply in some circumstances …
    - `Max transaction fee, if all above thingies apply: 5.4% + 0.30`

Also since we're thinking about hierarchy, the total fees line is maybe more important than the other info, so maybe it should be bigger / bolder or somehow separated.

**Comment by aheckler** [comment]
The reason I suggested the popup contain the base fee, intl. payment fee, FX fee, and dispute fee is because I tend to think of those as the 4 "basic" fees. It's how [our fees page](https://woocommerce.com/document/woopayments/fees-and-debits/fees/) is organized after all (see the table columns).

If we're trying to show a maximum fee, there are some one-offs we'd need to take into account. For example, the +1% fee on any order containing a subscription renewal or initial order, but only if you're using Stripe Billing (still available!).

IMHO, less is more here, and perhaps we could also consider not showing any more fee details in the popup, and just saying something like this instead:

> The exact fee for each order depends on various factors, like the customer currency and your account country. Learn more about WooPayments fees in your country.

**Comment by haszari** [comment]
Maybe just a min-max range and link to the "fine print".

It feels like we need the max close by since the main ui shows the minimum, so maybe the primary purpose of the tooltip is to illustrate the pessimistic case / max fees.

**Comment by pierorocca** [comment]
More than likely once the new onboarding flow is in place, we'll link out to pricing or make it easy to find pricing rather than having pricing always front and center on every interaction.

**Comment by pierorocca** [comment]
In the short term the tooltip could simply contain "Learn more about WooPayments fees in your country." It's hard to describe the complexity of fees in that small of a space and IMO better for a user to see all fees in a single table than have to traverse every tooltip. And it's good practice to have a single source of truth for this information.

Then once the new onboarding flow is in place which links to pricing, we can remove the pills and have a single link somewhere to fees.

**Comment by rogermattic** [comment]
Hey! This was the last design iteration I made to improve this tooltip as per p6riRB-a76 :

![image](https://github.com/user-attachments/assets/227058e7-620c-429b-92e0-de340c154d75)

**Comment by haszari** [comment]
We have an improved design from @rogermattic (`Regular rates`) which would address the concerns raised – let's implement it!

![Image](https://github.com/user-attachments/assets/9837e1d3-fc0b-4ccf-a76a-49eab0829dde)

Checking with various stakeholders – @aheckler @pierorocca  are you happy for the above design to be implemented? Note I'm excluding the `Discounted rates` customised tooltip for now, we can treat that as a follow up enhancement. 

Marking as `medium` priority, would be nice to tidy this up.

**Comment by aheckler** [comment]
LGTM!

**Comment by haszari** [comment]
Note – I've changed the focus area from `Fee & VAT invoices` to `positioning & fees`, as this is better match.

**Comment by pierorocca** [comment]
There is not alignment in p6riRB-a76-p2 as far as I can tell. I'm supportive of removing the pricing pills, but the modal approach is a no go for the amount of noise and prompts it generates and because the modal is important to reserve for future use. Additionally I don't quite get the modal on checkbox, with an Activate CTA, than then requires a "Save" action to actually save. It's not a fully baked solution. It's a lot of road bumps for getting to yes.

**Comment by rogermattic** [comment]
As I suggested in my comment under p6riRB-a76#comment-11338 I think it's worth splitting the discussion on the activation flow (modal approach etc.) and the implementation of the improved pills (adding an ( i ) icon to the activated state, with the improved toggle tip).

**Comment by pierorocca** [comment]
@rogermattic the latest proposal p6riRB-a76#comment-11338 in your design works for me. It achieves all of the objectives I had of avoiding new steps, not impacting attach rates, and not creating price sensitivity. I think this is a very good option for the current settings design.

**Comment by haszari** [comment]
@pierorocca @rogermattic could you paste the relevant design / plan / approach in this issue directly so it's easily actionable for an engineer to pick up? Could even update the description. The back & forth on this issue and links to internal P2s is getting hard to follow – but now we have an actionable plan 🎉🙌

**Comment by rogermattic** [comment]
Great @pierorocca ! 

@haszari I'll sort it out!

Edit: Description updated and the P2 is published here: p6riRB-b7J

**Comment by mjdeacon** [comment]
Hey @aheckler do we have user complaints with our current design and which OKR does this align to best? If this is coming into the billing domain to deliver, we need to prioritise it against all the other work we need to deliver for V2 and Adyen.

**Comment by aheckler** [comment]
> do we have user complaints with our current design

Pinging some folks who are in a better position than me to answer this: @AashikP @francoishvz @csmcneill. 

That said, even if we have literally _zero_ complaints about this, I still think we should fix it, given that the existing toast/popup thing is so plainly misleading:

![Screenshot taken on 2024-09-19 at 11 11 45 UTC@2x](https://github.com/user-attachments/assets/0ae62aaf-d407-4ea6-9f20-ea34e0af2ec6)

The main issue is the tooltip (as a whole) doesn't clearly indicate what is always charged vs. what might be charged under specific conditions.

A part of this is the "Total per transaction" line, which indicates the maximum fees that can be charged on a per-transaction basis. However, the current tooltip design/copy doesn't provide any indication that these additional fees are not always be charged (e.g., domestic transactions with no FX).

**Comment by AashikP** [comment]
As for user complaints specific to fee, but not specifically about this tooltip, one has been reported here: https://github.com/Automattic/woocommerce-payments/issues/9422

I'll let @francoishvz  and rest of the team chime in, in case I'm missing something.

> That said, even if we have literally zero complaints about this, I still think we should fix it, given that the existing toast/popup thing is so plainly misleading:

💯

**Comment by mjdeacon** [comment]
I'm not disagreeing there are improvements that could be made to make it clearer but I don't think it's misleading. Trying to balance this request with other work we've trying to deliver. 

Are we also considering moving pricing information out of the plug in into support pages for v2? If so, what would be the value of this and how long would it live for?

**Comment by rogermattic** [comment]
@mjdeacon  when I first came across that this is something we could potentially improve I worked with @kalessil – perhaps he can add some insight here.

> Are we also considering moving pricing information out of the plug in into support pages for v2? 

Not sure we are. There are arguments for keeping this (in this non-The correct spelling is "intrusive." form as we're proposing now) and removing it, it surely sparked a discussion between @pierorocca and @timmy5685 and me under p6riRB-a76-p2

Personally, I do think it's the least we can do to guide the user and connect the WooCommerce dashboard with the pricing information in the support docs that we have.

**Comment by aheckler** [comment]
> Are we also considering moving pricing information out of the plug in into support pages for v2?

Full pricing info is indeed available in our docs [here](https://woocommerce.com/document/woopayments/fees-and-debits/fees/).

**Comment by pierorocca** [comment]
It's a good point. Pricing could vary by partner with Adyen. Hopefully the pricing is merchant specific not a generic price by country?

**Comment by csmcneill** [comment]
Currently, displayed pricing is merchant-specific per https://github.com/Automattic/woocommerce-payments/pull/5299

**Comment by flakas** [comment]
For context, we split this issue up into two on Linear:
* https://linear.app/a8c/issue/WOOPMNT-4279 for only the tooltip revision
* https://linear.app/a8c/issue/WOOPMNT-6339/payments-settings-remove-fee-pills-show-fee-help-icon-only-on for the broader redesign
